4MD/src/main/resources/templates/add-recipe.html

```diff
@@ -0,0 +1,168 @@
+            </div>
+            <div>
+                <label class="form-label">Dish Type</label>
+                <select class="form-select" required th:field="*{dishType}" name="dishType">
+                    <option selected disabled value="">Choose...</option>
+                    <option th:each="enumName : ${T(com.infoshareacademy.four_md.model.DishType).values()}"
+                            th:value="${enumName}" , th:text="${enumName}"></option>
+                </select>
+            </div>
+
+            <div>
+                <label class="form-label">Difficulty</label>
+                <select class="form-select" required th:field="*{difficulty}" name="difficulty">
+                    <option selected disabled value="">Choose...</option>
+                    <option th:each="enumName : ${T(com.infoshareacademy.four_md.model.Difficulty).values()}"
+                            th:value="${enumName}" , th:text="${enumName}"></option>
+                </select>
+            </div>
+
+            <div class="shadow p-3 bg-body rounded" >
+            <div id="ingredientToCopy">
+                <h4 class="text-center mt-3">Ingredient</h4>
+                <div class="mb-3">
+                    <label class="form-label">Name</label>
+                    <input class="form-control" required type="text" placeholder="Name"
+                           th:field="*{ingredientsList[0].name}">
+                </div>
+                <div class="mb-3">
+                    <label class="form-label">Quantity</label>
+                    <input type="number" step="any" class="form-control" required th:placeholder="'Quantity'"
+                           th:field="*{ingredientsList[0].quantity}">
+                </div>
+                <div class="mb-3">
+                    <label class="form-label">Unit</label>
+                    <select class="form-select" required th:field="*{ingredientsList[0].unit}">
+                        <option selected disabled value="">Choose...</option>
+                        <option th:each="enumName : ${T(com.infoshareacademy.four_md.model.Unit).values()}"
+                                th:value="${enumName}" , th:text="${enumName}"></option>
+                    </select>
+                </div>
+                <div class="mb-3">
+                    <label class="form-label">Price</label>
+                    <div class="input-group">
+                        <input type="number" step="any" class="form-control" required th:placeholder="'Cost'"
+                               th:field="*{ingredientsList[0].price}" name="price">
+                        <span class="input-group-text">PLN</span>
+                    </div>
+                </div>
+            </div>
+
+                <div class="d-grid mt-3 mb-3 col-5">
+                    <a class="btn btn-outline-primary" role="button" id="btn2">+ Add new Ingredient</a>
```

Comment:
Jezeli dajesz mozliwosc dodawadia dodatkowych przepisow, to moze daj tez mozliwosc ich usuwania ?!?